src/D_functions_and_procedures.sql

```diff
@@ -146,3 +146,53 @@ BEGIN
     PERFORM rate(john_id, 'tt1375666', 9.0); 
     PERFORM rate(jane_id, 'tt1375666', 7.0); 
 END $$;
+
+
+-- D6 View to simplify queries for actors and their associated media
+CREATE OR REPLACE VIEW actor_media_view AS
+SELECT 
+    p.person_id,
+    p."name" AS actor_name,
+    p.imdb_id,
+    cm.media_id,
+    cm."role",
+    cm."character"
+FROM 
+    person p
+JOIN 
+    cast_member cm ON p.person_id = cm.person_id;
+
```

Comment:
Not sure if it is necessary to create a view for this function. Probably using a WITH statement can be used within the function instead of creating a view in the whole DB.
I have checked what is a difference between it is up to you to decide

<img width="720" alt="image" src="https://github.com/user-attachments/assets/82bfb1a0-9135-49cf-8240-684132827788">

Otherwise the function seems cool to me! :D

Reply:
Is we do not need a view at all, I guess I thought that it would be nice considering the note from the task:

*Hint: You may for this as well as for other purposes find a view helpful to make query
expressions easier (to express and to read). An example of such a view could be one that
collects the most important columns from title, principals and name in a single virtual table.*

So I think we should keep it like this, if there is no obstructions 😉